compiler/src/main/scala/org/scalajs/nscplugin/GenJSCode.scala

```diff
@@ -5451,8 +5508,8 @@ abstract class GenJSCode[G <: Global with Singleton](val global: G)
 
         wasRepeated match {
           case Some(true) =>
-            reversedArgs =
-              genPrimitiveJSRepeatedParam(arg) reverse_::: reversedArgs
+            reversedArgs = genPrimitiveJSRepeatedParam(
+                arg) reverse_::: reversedArgs
```

Comment:
Other example of bad assignment?

Reply:
have you tried https://scalameta.org/scalafmt/docs/configuration.html#newlinesafterinfix?